### PR TITLE
Regular updates

### LINKED ISSUE #1503: Integration tests on Rococo handles/
Integration tests on Rococo handles/

https://github.com/LibertyDSNP/frequency/tree/main/integration-tests/handles

These should run both rococo and local. 
If we think we should skip any of these it will be up to the developers to make the judgment call on how valuable it would be to run on Rococo.

### PR DESCRIPTION
**Pull request by wilwade**
# Goal
The goal of this PR is to fix various dependabot issues as well as other various updates

# Discussion
- Update api-augment packages
- Update integration tests packages
- Update integration tests to use Helia instead of js-ipfs
- Cargo updates
- Closes #1503
- Closes: #1663

### CODE REVIEW

File: integration-tests/.relay-chain.mocharc.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     ],
     "loader": "ts-node/esm",
     "spec": [
-        "./{,!(node_modules|load-tests)/**}/{createMsa,addIPFSMessage,staking,transactions}.test.ts"
+        "./{,!(node_modules|load-tests)/**}/{createMsa,addIPFSMessage,staking,transactions,handles}.test.ts"
```
Comment (wilwade) [inline integration-tests/.relay-chain.mocharc.json:13]: Added support for the handles test in the testnet test runner

File: integration-tests/messages/ipfs.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { unixfs } from '@helia/unixfs';
+import { createHelia } from 'helia';
+
+export async function ipfsCid(content: string, path: string) {
+  // create a Helia node
+  const helia = await createHelia({ start: false });
```
Comment (wilwade) [inline integration-tests/messages/ipfs.ts:6]: We don't actually need to start the server to get the CID from it

File: integration-tests/messages/ipfs.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { unixfs } from '@helia/unixfs';
+import { createHelia } from 'helia';
```
Comment (wilwade) [inline integration-tests/messages/ipfs.ts:2]: js-ipfs is deprecated. Helia is now the standard.

File: integration-tests/package.json [hunk 4]
```diff
@@ -2,12 +2,13 @@
     "name": "frequency-integration-tests",
     "version": "1.0.0",
     "description": "Integration tests for the Frequency project",
+    "type": "module",
```
Comment (wilwade) [inline integration-tests/package.json:5]: Integration tests are now esmodules

File: integration-tests/tsconfig.json [hunk 5]
```diff
@@ -4,7 +4,7 @@
     "compilerOptions": {
         "allowSyntheticDefaultImports": true,
         "esModuleInterop": true,
-        "module": "CommonJS",
+        "module": "esnext",
```
Comment (JoeCap08055) [inline integration-tests/tsconfig.json:7]: ❤️

**Review by JoeCap08055 — APPROVED** [review]
lgtm

File: integration-tests/messages/ipfs.ts (hunk 3 above)
Comment (saraswatpuneet) [reply integration-tests/messages/ipfs.ts:2]: did you figure out helia? would be great if we can get a refresher

Comment (wilwade) [reply integration-tests/messages/ipfs.ts:2]: Mostly just used https://github.com/ipfs/helia/wiki/Migrating-from-js-IPFS and the other Helia examples.

**Pull request merged**